common/buildcraft/BuildCraftTransport.java

```diff
@@ -676,11 +676,11 @@ public void loadRecipes() {
 				String dye = ColorUtils.getOreDictionaryName(15 - i);
 				GameRegistry.addRecipe(new ShapedOreRecipe(new ItemStack(lensItem, 8, i),
 						"OSO", "SGS", "OSO",
-						'O', "ingotIron", 'S', dye, 'G', "blockGlass"
+						'O', "ingotIron", 'S', "dye", 'G', "blockGlass"
 						));
 				GameRegistry.addRecipe(new ShapedOreRecipe(new ItemStack(lensItem, 8, i + 16),
 						"OSO", "SGS", "OSO",
-						'O', Blocks.iron_bars, 'S', dye, 'G', "blockGlass"
+						'O', Blocks.iron_bars, 'S', "dye", 'G', "blockGlass"
```

Comment:
if you change this then get it out of the for loop as you are now registering the same recipe multiple times